src/Serializers/ContractBytecodeSerializer.js

```diff
@@ -0,0 +1,34 @@
+const RLP = require('rlp')
+const BaseSerializer = require('./BaseSerializer')
+const BytecodeSerializer = require('./BytecodeSerializer')
+const {byteArray2Int, byteArray2Hex} = require('../utils/int2ByteArray')
+const {decode} = require('../utils/RLPInt')
+
+class ContractBytecodeSerializer extends BaseSerializer {
+    constructor(globalSerializer) {
+        super()
+        this._bytecodeSerializer = new BytecodeSerializer(globalSerializer)
+    }
+
+    deserializeStream(data) {
+        const buffer = new Uint8Array(data)
+        const [byteArraySize, remainder] = decode(buffer.slice(2))
+        const overallSize = Number(byteArraySize) + 64 // TODO: figure out where comes 64
```

Comment:
`RLP.decode` just fails on any other length, I also tested it with a payload of a different size

Reply:
docs?

Reply:
https://github.com/aeternity/protocol/blob/c0d7455/serializations.md#contractbytearray
https://github.com/aeternity/protocol/blob/c0d7455/serializations.md#integer
Gaith helped me understand